package.json

```diff
@@ -11,6 +11,10 @@
     "bin",
     "lib"
   ],
+  "engines" : {
+    "node" : ">=4.2.0 <5.0.0 || >=6.9.0",
+    "npm": ">=2.14.7"
```

Comment:
I think we want to go with simply doing >4 for node, and we don't require any special npm version, so we also do not need to put a npm line here.